app/helpers/smart_proxies_helper.rb

```diff
@@ -1,19 +1,54 @@
 module SmartProxiesHelper
   def proxy_actions(proxy, authorizer)
-    [if proxy.has_feature?('Puppet CA')
-       [display_link_if_authorized(_("Certificates"), hash_for_smart_proxy_puppetca_index_path(:smart_proxy_id => proxy).
-                                merge(:auth_object => proxy, :permission => 'view_smart_proxies_puppetca', :authorizer => authorizer)),
-        display_link_if_authorized(_("Autosign"), hash_for_smart_proxy_autosign_index_path(:smart_proxy_id => proxy).
-                                merge(:auth_object => proxy, :permission => 'view_smart_proxies_autosign', :authorizer => authorizer))]
-     end,
+    [ display_link_if_authorized(_("Edit"), hash_for_edit_smart_proxy_path(:id => proxy), :class => 'edit_two_pane'),
```

Comment:
Why removing all of the previous options? I saw them under 'Select action' on `smart_proxies/:id` show, but now something that used to take me 1 click now is 2 clicks away.